### PR TITLE
Add notes for Laravel Sail users 

### PR DESCRIPTION
**Pull request by abrardev99**
Related issue [Tenancy for Laravel & Laravel Sail ](https://github.com/archtechx/tenancy/issues/691)

This PR adds the notes for Laravel sail users to grant `create` permissions database permissions to `sail` MySQL users. 

I've added notes on pages:
[Quickstart](https://tenancyforlaravel.com/docs/v3/quickstart/#creating-tenants)
image

[Configuration](https://tenancyforlaravel.com/docs/v3/configuration#database) 
image

### CODE REVIEW

File: source/docs/v3/configuration.blade.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ The list of domains that host your [central app]({{ $page->link('the-two-applica
 This config array lets you enable, disable or add your own [tenancy bootstrappers]({{ $page->link('tenancy-bootstrappers') }}).
 
 ### Database {#database}
+> Note: If you're using Laravel [Sail](https://laravel.com/docs/9.x/sail), ensure that `DB_USERNAME` has the necessary permissions to create databases. You can grant access to users by running the following command in MySQL console `grant create on *.* to 'sail'@'%';`.
```
Comment (lukinovec) [inline source/docs/v3/configuration.blade.md:63]:
```suggestion
> Note: If you're using [Laravel Sail](https://laravel.com/docs/9.x/sail), ensure that `DB_USERNAME` has the necessary permissions to create databases. You can grant access to users by running `grant create on *.* to 'sail'@'%';` in the MySQL console.
```

File: source/docs/v3/quickstart.blade.md [hunk 2]
```diff
@@ -176,6 +176,8 @@ To have users in tenant databases, let's move the `users` table migration (the f
 
 ## Creating tenants {#creating-tenants}
 
+> Note: If you're using Laravel [Sail](https://laravel.com/docs/9.x/sail), ensure that `DB_USERNAME` has the necessary permissions to create databases. You can grant access to users by running the following command in MySQL console `grant create on *.* to 'sail'@'%';`.
```
Comment (lukinovec) [inline source/docs/v3/quickstart.blade.md:179]:
```suggestion
> Note: If you're using [Laravel Sail](https://laravel.com/docs/9.x/sail), ensure that `DB_USERNAME` has the necessary permissions to create databases. You can grant access to users by running `grant create on *.* to 'sail'@'%';` in the MySQL console.
```

**Review by lukinovec — CHANGES_REQUESTED** [review]
I made the hyperlink cover "Laravel Sail" instead of just "Sail" because the name is not just "Sail", and I did a minor change in the MySQL command suggestion. Other than that, the notes look good.

**Comment by stancl** [comment]
Are all reviews resolved now? @lukinovec can approve the changes if the PR is fine now

**Pull request merged**